machine_management/control_plane_machine_management/cpmso_provider_configurations/cpmso-config-options-azure.adoc

```diff
@@ -1,74 +1,27 @@
 :_mod-docs-content-type: ASSEMBLY
-[id="cpmso-config-options-azure"]
-= Control plane configuration options for Microsoft Azure
 include::_attributes/common-attributes.adoc[]
+[id="cpmso-config-options-azure"]
+= Control plane configuration options for {azure-full}
```

Comment:
Just an FYI that for the Azure, CGP, and Nutanix versions of this assembly, there is a discrepancy between the assembly titles per the topic map and the titles per the assembly heading (i.e. they're missing the "Changing" gerund at the beginning):
```suggestion
= Changing control plane configuration options for {azure-full}
```

Come to think of it though, the "Control plane options for [platform]" assemblies seem to only have conceptual content, so maybe it's all the other assemblies that should be changed to not have the "changing" gerund at the beginning?

Either way, I'd just make sure there's consistency between the topic map names and the actual assembly headings